src/Raven.Server/Documents/DocumentsChanges.cs

```diff
@@ -63,7 +63,7 @@ public void RaiseNotifications(DocumentChange documentChange)
 
             foreach (var connection in Connections)
             {
-                if (connection.Value.IsDisposed == false)
+                if (!connection.Value.IsDisposed)
```

Comment:
For reference, our convention is to use ` == false`

Reply:
Changed all usages of IsRaised to the implicit conversion and fixed all usages to obey to the coding convention